src/Controller/BackupDbController.php

```diff
@@ -21,20 +20,30 @@
  * @copyright  Softleister 2007-2021
  * @author     Softleister <info@softleister.de>
  *
- * @Route("/BackupDB", defaults={"_scope" = "frontend", "_token_check" = false})
+ * @Route("/BackupDB")
```

Comment:
_Note:_ I have removed these defaults as I don't think the make much sense for this controller. I don't think you are expecting `POST` requests to this controller, so I disabled the CSRF token check. Also this controller does not really render a Contao front end page, nor does it utilise anything from the Contao framework in terms of front end or back end and so I removed the scope as well.